modules/preview/preview.php

```diff
@@ -12,42 +12,30 @@
 
 use VIPWorkflow\Modules\Preview\PreviewEndpoint;
 use VIPWorkflow\VIP_Workflow;
-use VIPWorkflow\Modules\Shared\PHP\Module;
 use VIPWorkflow\Modules\Preview\Token;
+use WP_Query;
 
-class Preview extends Module {
-	public $module;
-
-	public function __construct() {
-		// Register the module with VIP Workflow
-		$this->module_url = $this->get_module_url( __FILE__ );
-
-		$this->module = VIP_Workflow::instance()->register_module( 'preview', [
-			'title'      => __( 'Preview Link', 'vip-workflow' ),
-			'module_url' => $this->module_url,
-			'slug'       => 'preview',
-			'autoload'   => true,
-		] );
-	}
-
+class Preview {
```

Comment:
For now, the `Preview` class does not extend `Module`. The base class adds what are essentially utility methods once the module loading system is removed. The preview module doesn't use any of those, and I think these could be relocated and called directly as a utility methods in other classes that do make use of them.

I'll wait until we migrate something that uses all of these (like `Custom_Status`) to make a judgement call there.